app/models/container_image.rb

```diff
@@ -59,5 +62,13 @@ def perform_metadata_scan(ost)
     scan_via_miq_vm(miq_cnt_group, ost)
   end
 
+  def tenant_identity
+    User.super_admin
```

Comment:
If we do not have tenant defined on the ContainerImage it should at least use the tenant of the `ext_management_system`.  

See host for an example: 
https://github.com/ManageIQ/manageiq/blob/74e99a3791b5d1b1ea5d756eb692ea4795a5af85/app/models/host.rb#L196